oml/lightning/callbacks/metric.py

```diff
@@ -72,11 +95,38 @@ def on_validation_batch_end(
             if is_last_expected_batch:
                 self.calc_and_log_metrics(pl_module)
 
+    def _log_images(self, pl_module: pl.LightningDataModule) -> None:
```

Comment:
This one must be a method of `MetricValCallback`
At least, because we may want to see images in vanilla python pipeline as well